draft-ietf-quic-version-negotiation.md

```diff
@@ -80,13 +80,33 @@ which leverages similarities between versions and can negotiate between the set
 of "compatible" versions without additional round trips.
 
 
-## Conventions and Definitions
+## Conventions
 
 {::boilerplate bcp14-tagged}
 
-In this document, the Maximum Segment Lifetime (MSL) represents the time a QUIC
-packet can exist in the network. Implementations can make this configurable, and
-a RECOMMENDED value is one minute.
+
+## Definitions
+
+The document uses the following terms:
+
+* In the context of a given QUIC connection, the "first flight" of packets
+  refers to the set of packets the client creates and sends to initiate the
+  connection before it has heard back from the server.
+
+* In the context of a given QUIC connection, the "client's chosen version" is
+  the QUIC version of the connection's first flight.
+
+* The "original version" is the QUIC version of the very first packet the client
+  sends to the server. If version negotiation spans multiple connections, the
```

Comment:
> spans multiple connections

Are you referring to the different initial flights between incompatible version negotiation? Do we really consider that multiple connections? I don't think it's ever been referred it this way before.

I think I understand what each of these terms mean, or at least what you're trying to define them as, but how they are worded here actually confuses me more.

Reply:
> Are you referring to the different initial flights between incompatible version negotiation? Do we really consider that multiple connections? I don't think it's ever been referred it this way before.

Yes, as described in the "Connections and Version Negotiation" section.

> I think I understand what each of these terms mean, or at least what you're trying to define them as, but how they are worded here actually confuses me more.

Can you help explain what you find confusing? It's not very actionable for us editors otherwise.

Reply:
AFAIU, there are not multiple connections involved here. The client is making one connection, that is going through the version negotiation process. I don't consider incompatible negotiation a new connection. When you say first flight, I believe you mean the very first flight, and then also when the client responds for an incompatible negotiation; but you that doesn't align with my understanding of this text: "before it has heard back from the server". If you're doing incompatible negotiation, that's after you've heard from the server.

Reply:
> AFAIU, there are not multiple connections involved here. The client is making one connection, that is going through the version negotiation process. I don't consider incompatible negotiation a new connection.

If you have thoughts on the "Connections and Version Negotiation" section, please take them to an issue instead of a comment thread on an editorial PR.

> When you say first flight, I believe you mean the very first flight, and then also when the client responds for an incompatible negotiation; but you that doesn't align with my understanding of this text: "before it has heard back from the server". If you're doing incompatible negotiation, that's after you've heard from the server.

I think that is covered by "In the context of a given QUIC connection". Am I missing something?

Reply:
A forward reference to the bit about connections and incompatible negotiation might help address some of this confusion.

Reply:
Good point, added